docs/developer-docs/multi-chain/chain-key-tokens/overview.mdx

```diff
@@ -19,6 +19,18 @@ Chain-key tokens can be swapped for one another seamlessly. Making transactions
 
 Each implementation of a chain-key token uses the [ICRC](https://github.com/dfinity/ICRC) token standard so that they can be easily integrated by services such as wallets and DEXs. To implement smart contracts that use chain-key tokens, simply make calls from a smart contract to the ICRC ledger for the chain-key token you wish to use. Using chain-key tokens does not require implementing a specific type of smart contract, and implementation is not restricted to a specific programming language.
 
+## Requirements for chain-key tokens
+
+In addition to the requirement that the chain-key token adheres to the  [ICRC](https://github.com/dfinity/ICRC) token standard, there are additional requirements to ensure a high degree of security and decentralization for all chain-key tokens:
+
+- The chain-key token must be a true digital "twin" in the sense that any user can convert the native token to the chain-key token and vice versa, paying only for the fees. This implies that the chain-key token must be fully backed by the corresponding native token at all times. Conversions may be delayed if one chain is temporarily unavailable but all conversions must be finalized eventually.
```

Comment:
> paying only for the fees.

What fees does this refer to, the transfer fees?

Reply:
Any fees such as transfer fees or conversion fees (in both directions).